kura/distrib/src/main/resources/common/comment-interfaces-file.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import subprocess
+import sys
+import shutil
+from os.path import exists
+
+LOG_MSG_PREFIX = "[comment_interfaces_file.py] "
+ETHERNET_TYPES = ['ethernet', 'eth', 'wired']
+WIRELESS_TYPES = ['wifi', 'wireless']
+INTERFACES_PATH = "/etc/network/interfaces"
+INTERFACES_OLD_PATH =  INTERFACES_PATH + ".old"
+INTERFACES_TMP_PATH = INTERFACES_PATH + ".tmp"
+
+def get_eth_wlan_interfaces_names():
+    """Reads the network interface names using 'nmcli dev' command.
+
+    It is assumed that at least one ethernet interface is present.
+
+    Returns:
+        tuple of lists (eth_names, wlan_names) where:
+            'eth_names' are the found ethernet interface names ordered by name;
+            'wlan_names' are the found wireless interface names ordered by name, might be an empty list.
+    """
+    cmd_output = subprocess.check_output(['nmcli', 'dev']).decode(sys.stdout.encoding).strip()
+    # list comprehension to remove empty items
+    lines = [x.strip() for x in cmd_output.split('\n') if len(x.strip()) > 0]
+
+    # removing header
+    del lines[0]
+
+    inteface_names = list()
+
+    for line in lines:
+        row = [x.strip() for x in line.split(' ') if len(x.strip()) > 0]
+        
+        interface_name = row[0].lower()
+        interface_type = row[1].lower()
+
+        if interface_type in ETHERNET_TYPES or interface_type in WIRELESS_TYPES:
+            inteface_names.append(interface_name)
+        
+    inteface_names.sort()
+
+    print(LOG_MSG_PREFIX + 'Found interfaces: ', inteface_names)
+
+    return (inteface_names)
+
+def comment_paragraph(paragraph):
+    """Comment a paragraph
+
+    Returns:
+        a paragraph whose lines start with #
+    """
+    commented = ""
+    for line in paragraph.split("\n"):
+        if (not line.startswith("#")):
+            commented += "#" + line + "\n" 
+        else:
+            commented += line + "\n" 
+    commented += "\n"
+    return (commented)
+
+def main():
+    if (not exists(INTERFACES_PATH)):
+        print(LOG_MSG_PREFIX + "File " + INTERFACES_PATH + " does not exist.")
```

Comment:
A much less important note here (feel free to ignore this comment): you might want to have %-formatting here instead of relying on string concatenation. String concatenation might result in unexpected errors during type casting.

Example:

```suggestion
        print("%s File %s does not exist." % (LOG_MSG_PREFIX, INTERFACES_PATH))
```

Other ways to achieve the same thing:
- f-strings (introduced in Python 3.6)
- `str.format` function

Furthermore, instead of simply `print`-ing you can rely on the [built-in logger](https://docs.python.org/3/library/logging.html). It requires some setup but it is a built-in solution for this exact case.

Reply:
So... I'm a Python noob!